src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double sideLength;
+
+    public Square(Color color, double sideLength) {
+        super(color, Math.pow(sideLength, 2));
+        this.sideLength = sideLength;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: square, side: %f, radius: %f, color: %s\n",
+                area,
+                sideLength,
+                color
+        );
```

Comment:
The `draw` method should not be responsible for calculating the area. Consider overriding the `getArea` method from the `Figure` class and using it to get the area.